master/internal/trial.go

```diff
@@ -775,6 +776,11 @@ func (t *trial) killAndRemoveSocket(ctx *actor.Context, id cproto.ID) {
 // containerConnected message have been received from each container in the trial. The two messages
 // are not guaranteed to come in-order.
 func (t *trial) allReady(ctx *actor.Context) bool {
+	// If a trial has passed allReady it can never return to a state of not ready.
```

Comment:
question: Isn't the allReady tied to the lifetime of the containers in the trial, which should get reset when a trial fails and is restarted?

Reply:
Yep see L1068 of this change.

Reply:
done